vibe_code_apps_19/app/agent_api.py

```
@@ -737,12 +737,21 @@ def export_agent_bundle(
 
     # Analytic-tab CSVs (topology, data model, sensor health, RCx comfort, meters)
     try:
+        from app.data_contract import load_vav_to_ahu_map
         from app.data_model_tree import build_data_model_tree
 
+        vav_to_ahu: dict[str, str] = {}
+        if dataset.source_path:
+            try:
+                vav_to_ahu = load_vav_to_ahu_map(Path(dataset.source_path))
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)